[2021-12-13 14:25:36,551] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.pre_process manual__2021-12-13T14:25:31.976190+00:00 [queued]>
[2021-12-13 14:25:36,570] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.pre_process manual__2021-12-13T14:25:31.976190+00:00 [queued]>
[2021-12-13 14:25:36,571] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 14:25:36,571] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-13 14:25:36,571] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 14:25:36,591] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): pre_process> on 2021-12-13 14:25:31.976190+00:00
[2021-12-13 14:25:36,600] {standard_task_runner.py:52} INFO - Started process 9229 to run task
[2021-12-13 14:25:36,605] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'pre_process', 'manual__2021-12-13T14:25:31.976190+00:00', '--job-id', '1228', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpf0r8mgmz', '--error-file', '/tmp/tmptumzecl9']
[2021-12-13 14:25:36,606] {standard_task_runner.py:77} INFO - Job 1228: Subtask pre_process
[2021-12-13 14:25:36,720] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.pre_process manual__2021-12-13T14:25:31.976190+00:00 [running]> on host 218df5ddc363
[2021-12-13 14:25:36,795] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=pre_process
AIRFLOW_CTX_EXECUTION_DATE=2021-12-13T14:25:31.976190+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-13T14:25:31.976190+00:00
[2021-12-13 14:25:36,856] {python.py:152} INFO - Done. Returned value was: None
[2021-12-13 14:25:36,877] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=pre_process, execution_date=20211213T142531, start_date=20211213T142536, end_date=20211213T142536
[2021-12-13 14:25:36,938] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-13 14:25:37,021] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
